### ISSUE TITLE
Deleting and ordering

### ISSUE THREAD

**Issue by chtiland** [samwilson/dokuwiki-plugin-log404#6]
Hi !

When deleting an error, list order change, is there a way to fix this ?

It would be nice to have actions (Goto, delete, ignore) vertically aligned. 
Displaying errors in a table in a column and on the same row, in another column the actions (and IP in a specific column).

Thanks

**Comment by samwilson** [comment]
Do you mean the ordering of the top-level pages list (which should be by hit count, i.e. the first number on those lines) or the ordering of the hits within each page?

**Comment by mekineer** [comment]
Yes, when you click on "delete x log entries" on one of the items in the list, the page refreshes.  When it does, the order of the list changes, I guess randomly.
